src/components/data-table/_data-table-core.scss

```diff
@@ -63,6 +63,11 @@
     width: 100%;
   }
 
+  .#{$prefix}--data-table tbody tr,
+  .#{$prefix}--data-table tbody tr td {
+    transition: background-color $duration--fast-02 motion(standard, productive);
```

Comment:
Is this the background color of the rows changing on hover? If so, I think it can be snappier. Maybe `$duration--fast-01 motion(entrance, productive)`. Same style as buttons hovers.

Reply:
Done 👍